docs/design.adoc

```diff
@@ -314,3 +314,31 @@ liquidating deposits still attractive for arbitraging bots.
 |12 hours
 |Governance can chose which strategy should be used by tBTC v1 risk manager for
 depositing ETH signer bonds purchased from tBTC to the asset pool.
+|===
+
+== Notifier rewards
```

Comment:
Shouldn't it live under tBTC v1 Risk Manager?

Reply:
Addressed https://github.com/keep-network/coverage-pools/pull/111/commits/ef8b524c44f1ebac4bd0e5851ecb3d5ae5a2bef8